src/paperqa/readers.py

```diff
@@ -171,6 +173,53 @@ def parse_text(
     )
 
 
+def parse_office_doc(
+    path: str | os.PathLike,
+    page_size_limit: int | None = None,
+    **kwargs,
+) -> ParsedText:
+    """Parse office documents (.docx, .xlsx, .pptx) using unstructured, extracting text and images."""
+    elements = partition(str(path), **kwargs)
+
+    content_dict = {}
+    media_list = []
+    current_text = ""
+    media_index = 0
+
+    for el in elements:
+        if isinstance(el, Image):
+            if el.metadata.image_data:
+                image_data = el.metadata.image_data
+                # Create a ParsedMedia object
+                parsed_media = ParsedMedia(
+                    index=media_index,
+                    data=image_data,
+                    info={"suffix": el.metadata.image_mime_type},
+                )
+                media_list.append(parsed_media)
+                media_index += 1
+        elif isinstance(el, Table):
+            # For tables, we could get the HTML representation for better structure
+            if el.metadata.text_as_html:
+                current_text += el.metadata.text_as_html + "\n\n"
+        else:
+            current_text += str(el) + "\n\n"
+
+    # For office docs, we can treat the whole document as a single "page"
+    content_dict["1"] = (current_text, media_list)
+
+    return ParsedText(
+        content=content_dict,
+        metadata=ParsedMetadata(
+            parsing_libraries=["unstructured"],
```

Comment:
Can you include the version of `unstructured` here? Take a look at how the Docling reader does it (at `packages/paper-qa-docling`)